### ISSUE TITLE
Add a subforum for general mapping discussions

### ISSUE THREAD

**Issue by Xinnoh** [ppy/osu-web#2771]
There is nowhere to have a casual discussion about mapping on the forum

![screen shot 2018-03-22 at 9 36 44 pm](https://user-images.githubusercontent.com/30382015/37773661-2770b82e-2e19-11e8-9a1c-a149d51f36d7.png)

**Comment by peppy** [comment]
@Ephemeralis let's go through this at some point and reorganise the mapping areas of the forum. Feel free to add more information to this issues (I know there was already further discussion on discord which may be useful to append here).

**Comment by dorothy3242** [comment]
Did any discussion happen about this ever? It's badly needed.
Even if forums are less prominently used in the presence of discord, it is still a forward-facing public platform for mapping discussions. Having it so disorganised to the point of being unintuitive to use is unfriendly to new users.

### CURRENT PROBLEMS

> **Ranking Criteria is in the Development forum,** which is in the osu! forum
Yes RC is development, but this distances it from mapping and makes it less intuitive to discover.

> **Mapping techniques** in the G&R forum.
This subforum is unused due to mapping techniques on the wiki not being discussed anymore. Many users mistake this for being a general mapping forum due to it being towards the top of the forum page, and post there for help. Experienced mappers that know better don't check there, resulting in many threads where people don't get the help they need. 

>**Ranked and approved beatmaps**
It has no use anymore, besides the *Report a qualified beatmap* thread which currently has no where else to go in this horrible forum structure. That thread will need to be moved somewhere more intuitive and visible after the mapping forum is reorganised.

>**Pending Beatmaps**
Same issue as ranked & approved. It's been well over a year since the majority of maps have been moved to modding discussions, and the few mv1 left are either not going anywhere or can be easily ranked in modding discussions. Disqualifying mv1 beatmaps is not even supported, so it's nonsensical that they are still around instead of being moved.

>**WIP/Help**
The same issue as ranked and pending. However, the subforums of this forum are some of the most active mapping forums we have. The issue is the top-level of the forum is not in use. These subforums will need to be moved elsewhere, and the WIP forum itself  (beatmap threads) archived.
`General Questions` is used very similarly to `Mapping Techniques`. The `Storyboarding` forum is basically dead. The `Modding Queues` forum is great but could be better.

>**Beatmap graveyard**
archive this. seriously.

>**Beatmap projects**
An ok forum, but it's a bit disorganised. Contests, projects, and GD requests are mixed together. The GD requests drown out the contests and projects which are more likely to be checked regularly even if not frequently posted in.

>**Beatmap management**
Most of this takes place in the BN discord server now. The only function for this subforum is currently the cookie box. This could be repurposed in the forum reorganization.


OK. Now that I've extensively ranted about why the current forum organisation **sucks**, let's get on with my lengthy proposal for how to reorganize the forums.

### PROPOSAL

This is a detailed list of proposed forums and their functions, as well as how the current forums would be moved or merged to create them.

**Beatmap Thread Archive**
Archive beatmap threads from the ranked, pending, and wip beatmap forums in a read-only state. This should be available due to times when previous discussions on ranked maps may be researched.
Any remaining beatmaps that use forum modding should be automatically moved to modding discussions at this time.

**Beatmap Management Forum**
The top-level of this forum would allow any users to post in it. The `Ranking Criteria` forum would be merged into this one, and all ranking criteria discussion would happen here.
The thread from the current Beatmap management forum would be pinned here.
subforums:
- Nominator Discussion - Only BNs and higher power may post here. This subforum would be the same purpose as the current beatmap management forum. It still may not be used frequently, but it is still needed, hence it'd be a subforum. The current top-level of the `Beatmap Management` subforum would be moved here.
- Archive - The `Finalized/Denied Amendments` ranking criteria subforum and current Beatmap Management `Archive` subforum would both be here.

**General Mapping Discussion**
The top-level would allow any posts about anything related to mapping/storyboarding.
The current `General Questions` and `Mapping Techniques` and `Storyboarding` forums would be moved into this forum. This would unify what are currently extremely similarly used subforums.
Storyboarding subforum is either used for questions or guides, and not very frequently, so it would be a good idea to merge this into general mapping so long as a pinned thread clearly explains that storyboarding threads also go here.
Subforums:
- Guides - Allow for people to post guides and tutorial threads here. This would make them easier to find without requiring the use of a potentially easily outdated compendium thread.
- Contests - Contests would go here. Currently they go in `Beatmap Projects` and are easily drowned out by many GD request threads. This would make contests a bit more easily accessible and visible.
- Beatmap Projects - could either remain as a subforum or be merged into the general mapping discussion forum. It is still frequently used for gd requests.

**Modding Queues**
Perfectly fine forum on its own, it just doesn't need to be a subforum for the WIP forum which itself isn't used. Please see issue https://github.com/ppy/osu-web/issues/3899 for a proposal of various ideas in which modding queues functionality could be improved. 



I understand this initial proposal may not be perfect, but it's a start and can be worked on. So long as it actually gets worked on.

### TL;DR

- The current mapping forums suck and are all over the place.
- Suggestion:
Beatmap Archive
Beatmap Management forum - Discuss RC +bn app thread (has subforum for bn discussions)
General mapping discussion - General (has subforums for guides, contests, projects)
Modding queues

**Comment by FlyingStefan** [comment]
Bump as people constantly post semi-related to unrelated topics in Beatmap Projects that *are* okay technically but still have no place to go and ends up in an unfitting subforum. I'm on it to sort out things in the Beatmap Projects section but it's complicated to deal with things like ideas to map <genre> or specific types of maps without having a basis source.

**Comment by FlyingStefan** [comment]
@dorothy3242 I'm actually okay with contests being held in the beatmap projects forum but I understand the issue getting lost under the many GD requests.

I still think the BP forum as it is right now should still exist after a change and work similar in the way it currently does. There are lots of dead projects and while many of them are just not worth sorting out (or at least give an automated system like Graveyard or Archived for Help) it's better to dump them there and keep unspecified mapping-related threads in a separated forum.

**Comment by peppy** [comment]
Now that we finally finished deploying the new forum, this has been enacted. Hopefully all bases have been covered. I didn't make the "contests" subforum just yet (it didn't look like there are many of these?) but feel free to try and change my mind.
